metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1087,10 +1106,60 @@ class MetalsLanguageServer(
             )
           )
         }.asJavaObject
+      case ServerCommands.StartDebugAdapter() =>
+        val args = params.getArguments.asScala
+        args match {
+          case Seq(param) =>
```

Comment:
We can match `Seq(params: JsonElement)` and then as[b.DebugSessionParams] will work without having to use implicit class on Any